### DISCUSSION TITLE
Linux build

### DISCUSSION THREAD

**Discussion by ghost** [aflacc/ProjectFNF#27] [General]
I'm going to attach Linux builds here when ProjectFNF gets updated, feel free to add to itch.io if you want.

**_Note: requires Lime to be installed as well as glibc 2.33 (2.33 is Ubuntu 21.04 or higher, not sure about other distros, check
 `ldd --version` to see if it will run_**

**Comment by ghost** [comment]
First release will be here in a minute, just building then will zip it

**Comment by ghost** [comment]
Built from latest GitHub source as of 20:30 BST May 4 2021
https://ufile.io/i03l7rpq

**Comment by aflacc** [comment]
my mans just being a complete second developer

**Reply by ghost** [reply]
Is that meant to be a good thing or a bad thing?

**Reply by ghost** [reply]
![image](https://user-images.githubusercontent.com/81651922/117333516-eb3b7c80-ae90-11eb-9d66-fcb80424ba48.png)
I just realised we're commenting from the future what

**Reply by aflacc** [reply]
dear god hes time traveling

**Reply by aflacc** [reply]
> Is that meant to be a good thing or a bad thing?

its a good thing

**Reply by ghost** [reply]
Nice

**Comment by ghost** [comment]
HECK YEAH WHOO

**Comment by ghost** [comment]
Latest Linux build here (git-31-may-1726.zip): https://ufile.io/b5als22w
(note: I can do Windows builds occasionally, just reply here when you want a Windows, Linux or both build)
Note: I am compiling to release preset

**Reply by aflacc** [reply]
If its just for downloads you can just check the itch.io I'll be putting stuff on there

**Comment by aflacc** [comment]
yo VMGuy28 would you be down to being able to put those on the itch.io page?

**Reply by ghost** [reply]
I don't have access, do I?

**Reply by aflacc** [reply]
what is your itch.io username

**Reply by ghost** [reply]
vmguy23
page is vmguy23.itch.io
